puppet/ha-puppet/screenshot.js

```diff
@@ -255,9 +325,31 @@ export class Browser {
             }
 
             const panel = panelResolver.children[0];
-            if (!panel) return false;
+            if (!panel) {
+              return false;
+            }
+            if (!("_panelState" in panel) || panel._panelState !== "loaded") {
+              return false;
+            }
 
-            return !("_loading" in panel) || !panel._loading;
+            const isLovelacePath = window.location.pathname.startsWith("/lovelace");
```

Comment:
This is not guaranteed, if you make a second dashboard, it won't start with `lovelace`